test/integration/golden_response/staging/get_property_values/contained_in_place.json

```diff
@@ -4,39 +4,39 @@
       {
         "dcid": "geoId/0677000",
         "name": "Sunnyvale",
-        "provenanceId": "dc/sm3m2w3",
+        "provenanceId": "dc/5n63hr1",
```

Comment:
For all places, WikidataOtherIdGeos  should generally be the provenance because it includes a a long list of ID properties, and the choice is based on most PVs.